src/common/pal/README.md

```diff
@@ -0,0 +1,52 @@
+# Platform Abstraction Layer (PAL)
+
+The aim of this module is to provide a the tools necessary to independize wazuh agent's code from the platform it will be compile for.
```

Comment:
```suggestion
The aim of this module is to provide the tools necessary to independize wazuh agent's code from the platform it will be compile for.
```